Modules/Sources/WordPressUI/Extensions/UIColor+Helpers.swift

```diff
@@ -0,0 +1,108 @@
+import UIKit
+
+public extension UIColor {
+
+    /// Creates a color based on a hexString. If the string is not a valid hexColor it return nil
+    /// Example of colors: #FF0000, #00FF0000
+    ///
+    /// This method should only be used for parsing colors from third-party content  – if you're defining a color in code, use `fromHex`
+    convenience init?(hexString: String) {
+        let hex = hexString.trimmingCharacters(in: CharacterSet.alphanumerics.inverted)
+
+        guard let int = Scanner(string: hex).scanUInt64(representation: .hexadecimal) else {
+            return nil
+        }
+
+        let a, r, g, b: UInt64
+        switch hex.count {
+        case 3: // RGB (12-bit)
+            (a, r, g, b) = (255, (int >> 8) * 17, (int >> 4 & 0xF) * 17, (int & 0xF) * 17)
+        case 6: // RGB (24-bit)
+            (a, r, g, b) = (255, int >> 16, int >> 8 & 0xFF, int & 0xFF)
+        case 8: // ARGB (32-bit)
+            (a, r, g, b) = (int >> 24, int >> 16 & 0xFF, int >> 8 & 0xFF, int & 0xFF)
+        default:
+            return nil
+        }
+        self.init(red: CGFloat(r) / 255, green: CGFloat(g) / 255, blue: CGFloat(b) / 255, alpha: CGFloat(a) / 255)
+
+    }
+
+    convenience init(fromHex hex: UInt32) {
+        self.init(
+            red: CGFloat(UInt8((hex & 0xFF0000) >> 16)) / 0xFF,
+            green: CGFloat(UInt8((hex & 0xFF00) >> 8)) / 0xFF,
+            blue: CGFloat(UInt8(hex & 0xFF)) / 0xFF,
+            alpha: 1.0
+        )
+    }
+
+    convenience init(fromHex hex: UInt32, alpha: CGFloat) {
+        self.init(
+            red: CGFloat(UInt8((hex & 0xFF0000) >> 16)) / 0xFF,
+            green: CGFloat(UInt8((hex & 0xFF00) >> 8)) / 0xFF,
+            blue: CGFloat(UInt8(hex & 0xFF)) / 0xFF,
+            alpha: alpha
+        )
+    }
+
+    /// Produces an 6-digit CSS color hex string representation of `UIColor`.
+    var hexString: String {
+        guard let components = self.cgColor.components else {
+            preconditionFailure("Color can't be represented in hexadecimal form")
+        }
+
+        if components.count == 2 {
+            let w = Float(components[0])
```

Comment:
(nit) a bit odd to ignore the `alpha` component by default, but since it's a simple conversion from Objective-C, it's OK